docs/en/relying-party-solution.rst

```diff
@@ -16,139 +16,123 @@ This section describes how a Relying Party may ask to a Wallet Instance the pres
 
 In this section the following flows are described:
 
-- **Remote Same Device Flow**, where the Verifier and the Wallet Instance acts in the same device.
-- **Remote Cross Device Flow**, where the Verifier and the Wallet Instance acts in different devices and the Verifier is a remote Relying Party.
+- **Remote Same Device Flow**, where the Web Browser and the Wallet Instance are used in the same device.
```

Comment:
Couldn't it also be an app? I propose a generic name

Reply:
I suppose that even if embedded in the app it is always a web browser (better: user agent)

may we say just user-agent?

Reply:
better!

Reply:
In some contexts it's used the term `Reader Instance` as the _abstract_ agent that performs a single interaction with the Relying Party (that is: a Reader Instance relates to Relying Party as Wallet Intance relates to Wallet Provider).

Reply:
Reader Instance is defined for proximity flows while in this section we only cover remote flows

Reply:
I see. It's just a proposal to use a name that does not bring any technological constraint/expectation. User-agent is fine; in same-device flow, we can have web applications (browser) or native applications.